### PR TITLE
Annagrin/prep to publish dwds

### PR DESCRIPTION
**Pull request by annagrin**
Prepare to publish dwds 13.0.0

### CODE REVIEW

File: webdev/pubspec.yaml [hunk 1]
```diff
@@ -48,9 +48,9 @@ dev_dependencies:
   webdriver: ^3.0.0
 
 # Uncomment for development versions
-# dependency_overrides:
-#   dwds:
-#     path: ../dwds
+dependency_overrides:
```
Comment (elliette) [inline webdev/pubspec.yaml:51]: Shouldn't this be commented out?

File: dwds/test/chrome_proxy_service_test.dart [hunk 2]
```diff
@@ -1368,8 +1368,8 @@ void main() {
       var resolvedUris =
           await service.lookupResolvedPackageUris(isolateId, uris);
 
-      var packageUris = await service.lookupPackageUris(
-          isolateId, resolvedUris.uris as List<String>);
+      var packageUris =
+          await service.lookupPackageUris(isolateId, resolvedUris.uris);
```
Comment (kevmoo) [inline dwds/test/chrome_proxy_service_test.dart:1372]: Oh! I just sent a PR to fix this. 😄

Comment (elliette) [inline dwds/test/chrome_proxy_service_test.dart:1372]: Why is this necessary?

Comment (kevmoo) [reply dwds/test/chrome_proxy_service_test.dart:1372]: Either a refinement in static analysis in the latest dev SDK (released hours ago) or changes in the api for `.uris` w/ a dot release of analyzer.

Comment (kevmoo) [reply dwds/test/chrome_proxy_service_test.dart:1372]: https://github.com/dart-lang/webdev/runs/5499790417?check_suite_focus=true#step:7:11

Comment (kevmoo) [reply dwds/test/chrome_proxy_service_test.dart:1372]: landed.

File: webdev/pubspec.yaml (hunk 1 above)
Comment (annagrin) [reply webdev/pubspec.yaml:51]: We actually need both types of testing:
- Current dwds to make sure current changes do not break webdev (that's what I would like to test now), 
- dwds from pub to make sure current webdev works with released dwds.

We switch to dwds from pub right after dwds is released to release webdev, and then back to local. 
In the long term, I think we should always have local here and have a separate release test suite that might only run when needed.

File: dwds/test/chrome_proxy_service_test.dart (hunk 2 above)
Comment (annagrin) [reply dwds/test/chrome_proxy_service_test.dart:1372]: thanks!

Comment (annagrin) [reply dwds/test/chrome_proxy_service_test.dart:1372]: No its package:vm_service update that caused it, type change in `UriList.uris`

**Comment by annagrin** [comment]
I'll keep this PR waiting for a little bit  - discovered some extra printing that now shows in flutter tools, need to send a PR first to mend it.

File: webdev/pubspec.yaml (hunk 1 above)
Comment (elliette) [reply webdev/pubspec.yaml:51]: Ok! Do you mind opening a PR to include that in the release instructions here: https://github.com/dart-lang/webdev/blob/master/dwds/CONTRIBUTING.md#step-2-publish-dwds-to-pub Thanks!
